spec/latest/superjson-spec.json

```diff
@@ -0,0 +1,186 @@
+{
```

Comment:
Can we have this as yaml as well? And I still wonder what should be source of truth. This or https://github.com/superfaceai/ast-js/blob/dev/src/interfaces/superjson/superjson.schema.json?

Reply:
I think I like the source of truth being the link you provided to the `ast-js` repo. What's the best path forward if we do that? Completely remove it and maybe provide a link? Git submodules? (hopefully not)

Reply:
see https://github.com/superfaceai/spec/pull/37#issuecomment-1060391474